docs/installation.md

```diff
@@ -8,10 +8,16 @@ kk create cluster --with-kubesphere nightly-$(date -d yesterday '+%Y%m%d')
 
 ## Install via helm
 
-First, please clone this git repository. Then run command: `make install-chart`
+First, add helm chart repo: `helm repo add ks https://kubesphere-sigs.github.io/ks-devops-helm-chart/`
```

Comment:
In this document, we name helm repo `ks`, but `ks-devops` is used in https://github.com/kubesphere-sigs/ks-devops-helm-chart#usage. That makes me confused. I think we should unify the names of two.

Reply:
So, what's your suggestion?

Reply:
I suggest using `ks-devops` as helm repo name. Please see https://openebs.github.io/charts/.

Reply:
Sure.